models/public/index.md

```diff
@@ -123,6 +123,21 @@ SSD-based and provide reasonable accuracy/performance trade-offs.
 | MobileFaceNet,ArcFace@ms1m-refine-v1 | [MXNet\*](./face-recognition-mobilefacenet-arcface/face-recognition-mobilefacenet-arcface.md) | face-recognition-mobilefacenet-arcface | 0.449 | 0.993 |
 | SphereFace                | [Caffe\*](./Sphereface/Sphereface.md) | Sphereface | 3.504 | 22.671 |
 
+## Human Pose Estimation Models
+
+Human pose estimation task is to predict a pose: body skeleton, which consists
+of keypoints and connections between them, for every person in an input image or
+video.  Keypoints are body joints, i.e. ears, eyes, nose, shoulders, knees, etc.
+There are two major groups of such metods: top-down and bottom-up.  The first
+detects persons in a given frame, crops or rescales detections, then runs pose
+estimation network for every detection. These methods are very accurate. The
+second finds all keypoints in a given frame, then groups them by person
+instances, thus faster than previous, because network runs once.
+
+| Model Name                | Implementation | OMZ Model Name | Accuracy | GFlops | mParams |
+| ------------------------- | ---------------| -------------- | -------- | ------ | ------- |
+| OpenPose                  | [TensorFlow\*](./openpose-cmu-tf/openpose-cmu-tf.md) | openpose-cmu-tf | 49.89 | 319.304 | 52.311 |
```

Comment:
why do you set accuracy in index.md and stay this field empty in the model config?
Is it made on some reason?